docs/topics_5/templates/document-attributes.adoc

```diff
@@ -35,11 +35,11 @@
 // * Doc metadata *
 // ****************
 :DocInfoProductName: Migration Toolkit for Applications
-:DocInfoProductNumber: 5.0
+:DocInfoProductNumber: 5.0.1
 :DocInfoProductNameURL: migration_toolkit_for_applications
 
 :OpenShiftProductNumber: 4.5
-:ProductVersion: 5.0.0
+:ProductVersion: 5.0.1
```

Comment:
The only impact this change will have in the user guide is in the name of the file to download to install for the IDE/Eclipse plug-in. Phil confirmed that this change was OK.

Reply:
Have you confirmed that the maxfiles value change is OK for 5.0.1?